js/huobi.js

```diff
@@ -2943,23 +2942,20 @@ module.exports = class huobi extends Exchange {
         params = this.omit (params, [ 'defaultSubType', 'subType' ]);
         const isolated = (marginMode === 'isolated');
         const cross = (marginMode === 'cross');
-        if (spot) {
-            if (isolated) {
-                method = 'spotPrivateGetV1MarginAccountsBalance';
-            } else if (cross) {
-                method = 'spotPrivateGetV1CrossMarginAccountsBalance';
+        const margin = (type === 'margin') || cross || isolated;
```

Comment:
@ttodua something is off here 
```
 p huobi balance
Python v3.10.9
CCXT v2.7.33
{}
huobi requires to release all resources with an explicit call to the .close() coroutine. If you are using the exchange instance with async coroutines, add `await exchange.close()` to your code into a place when you're done with the exchange and don't need the exchange instance anymore (at the end of your async coroutine).
```

Reply:
ah, forgot to place spot. please recheck now.
all I wanted to mean is that, if someone uses `fetchBalance ({marginMode: 'isolated/cross'})` I mean that user is requesting spot margin balance (isolated/cross). is not that correct expectation?

Reply:
hm, python version breaks, give me a min.

Reply:
the error seems to be happening in master too, without this PR.
```
cli.py huobi fetchBalance

Python v3.10.9
CCXT v2.7.33
Traceback (most recent call last):
  File "py\cli.py", line 232, in <module>
    asyncio.run(main())
  File "..\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "..\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "py\cli.py", line 109, in main
    keys = json.load(file)
  File "..\lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
  File "..\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 3901: character maps to <undefined>
```

Reply:
the last comment of mine turns out to be my local issue (another PR ready for that), but with this PR, i think there is no error at all. please re-check and let me know a bit more verbose log what/where you face issue.

Reply:
>  fetchBalance ({marginMode: 'isolated/cross'}) I mean that user is requesting spot margin balance 
@ttodua yes I think that is correct unless we have the ambiguity between isolated/cross on the swap wallet as well